eduNEXT/eox-nelp
# feat: add pearson_real_time_action which allows to execute rti reques…

## Description
This add a new action to the course enrollment admin panel that allows to execute the real_time_import_task  method with the given parameters 



## Testing instructions
1. Go to the course enrollment admin panel
2.  Select a user
3. Click the action dropdown and select the Execute RTI request option
4. Press go and check 
![image](https://github.com/eduNEXT/eox-nelp/assets/36200299/d99bede3-ea84-427f-b723-e506989822df)
### Before

### After

## Additional information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
- Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.

## Checklist for Merge
